staging/cse/windows/azurecnifunc.tests.suites/AzureCNI.Default.Overlay.conflist

```diff
@@ -0,0 +1,50 @@
+{
+  "cniVersion": "0.3.0",
+  "name": "azure",
+  "adapterName": "",
+  "plugins": [
+    {
+      "type": "azure-vnet",
+      "mode": "bridge",
+      "bridge": "azure0",
+      "executionMode": "v4swift",
+      "capabilities": {
+        "portMappings": true,
+        "dns": true
+      },
+      "ipam": {
+        "type": "azure-cns",
+        "mode": "v4overlay"
+      },
+      "dns": {
+        "Nameservers": [
+          "10.0.0.10",
+          "168.63.129.16"
+        ],
+        "Search": [
+          "svc.cluster.local"
+        ]
+      },
+      "AdditionalArgs": [
+        {
+          "Name": "EndpointPolicy",
+          "Value": {
+            "Type": "OutBoundNAT",
+            "ExceptionList": [
+              "10.240.0.0/16",
```

Comment:
overlay exception list should only contain cluster cidr

Reply:
Thank you @jaer-tsun . The UT read ```AzureCNI.Default.Overlay.conflist(mock the file download from https://acs-mirror.azureedge.net/azure-cni/.../10-azure.conflist)``` and run ```Set-AzureCNIConfig``` function which we want to test, then compare the result with ```AzureCNI.Expect.Overlay.conflist```.  So the ```AzureCNI.Expect.Overlay.conflist``` should only contain cluster cidr. Is it correct?